.github/workflows/update-ssdlc-report.yaml

```diff
@@ -40,79 +49,9 @@ jobs:
-          private-key: ${{ secrets.APIXBOT_APP_PEM }}
-      - name: Find JIRA ticket
-        id: find
-        uses: mongodb/apix-action/find-jira@3024080388613583e3bd119bfb1ab4b4dbf43c42
-        with:
-          token: ${{ secrets.JIRA_API_TOKEN }}
-          jql: project = CLOUDP AND status NOT IN (Closed, Resolved) AND summary ~ "Update Compliance Report"
-      - name: Set JIRA ticket (find)
-        if: steps.find.outputs.found == 'true'
-        run: |
-          echo "JIRA_KEY=${{steps.find.outputs.issue-key}}" >> "$GITHUB_ENV"
-      - name: Create JIRA ticket
-        uses: mongodb/apix-action/create-jira@3024080388613583e3bd119bfb1ab4b4dbf43c42
-        id: create
-        if: steps.find.outputs.found == 'false'
-        with:
-          token: ${{ secrets.JIRA_API_TOKEN }}
-          project-key: CLOUDP
-          summary: "[AtlasCLI] Update Compliance Report"
-          issuetype: Story
-          description: Update Compliance Report
-          components: AtlasCLI
-          assignee: ${{ secrets.ASSIGNEE_JIRA_TICKET }}
-          extra-data: |
-            {
-              "fields": {
-                "fixVersions": [
-                  {
-                    "id": "41805"
-                  }
-                ],
-                "customfield_12751": [
-                  {
-                    "id": "22223"
-                  }
-                ],
-                "customfield_10257": {
-                  "id": "11861"
-                }
-              }
-            }
-      - name: Set JIRA ticket (create)
-        if: steps.find.outputs.found == 'false'
-        run: |
-          echo "JIRA_KEY=${{steps.create.outputs.issue-key}}" >> "$GITHUB_ENV"
-      - uses: peter-evans/create-pull-request@271a8d0340265f705b14b6d32b9829c1cb33d45e
-        id: pr
-        with:
-          token: ${{ steps.app-token.outputs.token }}
-          committer: "${{ steps.app-token.outputs.user-name }} <${{ steps.app-token.outputs.user-email }}>"
-          author: "${{ steps.app-token.outputs.user-name }} <${{ steps.app-token.outputs.user-email }}>"
-          title: "${{ env.JIRA_KEY }}: Update compliance report for v${{ steps.extract.outputs.version }}"
-          commit-message: "${{ env.JIRA_KEY }}: Update compliance report for v${{ steps.extract.outputs.version }}"
-          delete-branch: true
-          base: master
-          branch: ${{ env.JIRA_KEY }}
-          labels: |
-            compliance
-            auto_close_jira
-          body: |
-            ## Proposed changes
-            Update compliance report for v${{ steps.extract.outputs.version }}
-            _Jira ticket:_ ${{ env.JIRA_KEY }}
-
-            Note: Jira ticket will be closed automatically when this PR is merged.
-
-      - name: Set auto merge
-        env:
-          GH_TOKEN: ${{ steps.app-token.outputs.token }}
-        run: |
-          gh pr merge "${{ steps.pr.outputs.pull-request-url }}" --auto --squash
+      - run: |
```

Comment:
nit: should we use a library instead? see https://github.com/mongodb/openapi/blob/7c80f06e5a6fd0066e3ba3623452b08fe7cb6c0b/.github/workflows/release-changelog.yml#L116-L118

Reply:
I would prefer not to add a third-party dependency. What is the value of it?